tests/network/vmi_slirp_interface.go

```diff
@@ -207,17 +205,11 @@ var _ = SIGDescribe("Slirp Networking", decorators.Networking, func() {
 			setDefaultNetworkInterface("bridge")
 		})
 		It("should reject VMIs with default interface slirp when it's not permitted", func() {
-			var t int64 = 0
-			vmi := tests.NewRandomVMI()
-			vmi.Spec.TerminationGracePeriodSeconds = &t
-			// Reset memory, devices and networks
-			vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("128Mi")
-			vmi.Spec.Domain.Devices = v1.Devices{}
-			vmi.Spec.Networks = nil
-			tests.AddEphemeralDisk(vmi, "disk0", v1.DiskBusVirtio, cd.ContainerDiskFor(cd.ContainerDiskCirros))
+			vmi := libvmi.NewGuestless()
```

Comment:
I wonder what does `Guestless` mean...

Reply:
That it has no OS.
If you have an idea for a better name, we can rename it.

Reply:
Do you mean that there is no boot source?  `NewWithoutBootSource`?

Reply:
I don't no if `WithoutBootSource` captures the fact that this VM is tiny and cannot really boot anything. I started with `Tiniest`; we can consider `WithoutGuest`, `WithoutOS`, `NoOS`. I don't like any of them in particular more than `Guestless`, but accept the verdict of the maintainers.